device.yml

```diff
@@ -419,4 +419,4 @@ groupMasks:
       SampleRate200Hz: 2
       SampleRate250Hz: 3
       SampleRate500Hz: 4
-      SampleRate1000Hz: 5
\ No newline at end of file
+      SampleRate1000Hz: 5
```

Comment:
I'm not sure what the change is here, maybe just a line ending or new line? @bruno-f-cruz if you can quickly double-check, it all looks good to me otherwise.

Reply:
Probably something that my editor picked up on. I forced-push the fix, looks ok now